lib/pos_consensus_controller.go

```diff
@@ -212,6 +212,82 @@ func (cc *ConsensusController) HandleBlock(pp *Peer, msg *MsgDeSoBlock) {
 	// TODO
 }
 
+// processNewTipBlock tries to apply a new tip block to both the Blockchain and FastHotStuffEventLoop data
```

Comment:
```suggestion
// tryProcessBlockAsNewTip tries to apply a new tip block to both the Blockchain and FastHotStuffEventLoop data
```
Starting comments that are immediately above function names allows goland to show this description as a preview when highlighting the function elsewhere in the codebase.